src/test/groovy/org/prebid/server/functional/tests/AliasSpec.groovy

```diff
@@ -0,0 +1,106 @@
+package org.prebid.server.functional.tests
+
+import org.prebid.server.functional.model.bidder.Generic
+import org.prebid.server.functional.model.request.auction.BidRequest
+import org.prebid.server.functional.service.PrebidServerException
+import org.prebid.server.functional.util.PBSUtils
+
+import static org.prebid.server.functional.model.bidder.BidderName.ALIAS
+import static org.prebid.server.functional.model.bidder.BidderName.GENERIC
+import static org.prebid.server.functional.model.bidder.CompressionType.GZIP
+import static org.prebid.server.functional.util.HttpUtil.CONTENT_ENCODING_HEADER
+
+class AliasSpec extends BaseSpec {
+
+    def "PBS should apply aliases for bidder when aliases corresponding to bidder request"() {
+        given: "Default bid request with alias"
+        def bidRequest = BidRequest.defaultBidRequest.tap {
+            ext.prebid.aliases = [(ALIAS.value): GENERIC]
+            imp[0].ext.prebid.bidder.alias = new Generic()
+        }
+
+        when: "PBS processes auction request"
+        def response = defaultPbsService.sendAuctionRequest(bidRequest)
+
+        then: "PBS contain two http calls and the same url for both"
+        def responseDebug = response.ext.debug
+        assert responseDebug.httpcalls.size() == 2
+        assert responseDebug.httpcalls[GENERIC.value]*.uri == responseDebug.httpcalls[ALIAS.value]*.uri
+
+        and: "Resolved request should contain aliases as in request"
+        assert responseDebug.resolvedRequest.ext.prebid.aliases == bidRequest.ext.prebid.aliases
+    }
+
+    def "PBS shouldn't apply aliases for bidder when unknown aliases corresponding to bidder request"() {
```

Comment:
The name differs from what you're testing. You defined a valid alias but didn't provide the config for it. But in the test name you are talking about unknown alias.